projects/packages/identity-crisis/src/class-identity-crisis.php

```diff
@@ -0,0 +1,1029 @@
+		if ( isset( self::$current_screen ) && 'toplevel_page_jetpack' !== self::$current_screen->id ) {
+			$classes .= ' is-dismissible';
+		}
+
+		if ( $this->admin_page_has_help_tabs() ) {
+			$classes .= ' has-help-tabs';
+		}
+		?>
+
+		<div class="<?php echo esc_attr( $classes ); ?>">
+			<?php $this->render_notice_header(); ?>
+			<div class="jp-idc-notice__content-header">
+				<h3 class="jp-idc-notice__content-header__lead">
+					<?php echo esc_html( $this->get_non_admin_notice_text() ); ?>
+				</h3>
+
+				<p class="jp-idc-notice__content-header__explanation">
+					<?php echo esc_html( $this->get_non_admin_contact_admin_text() ); ?>
+				</p>
+			</div>
+		</div>
+		<?php
+	}
+
+	/**
+	 * First "step" of the IDC mitigation. Will provide some messaging and two options/buttons.
+	 * "Confirm Staging" - Dismiss the notice and continue on with our lives in staging mode.
+	 * "Fix Jetpack Connection" - Will disconnect the site and start the mitigation...
+	 *
+	 * @return void
+	 */
+	public function display_idc_notice() {
+		$classes = 'jp-idc-notice inline notice notice-warning';
+		if ( $this->admin_page_has_help_tabs() ) {
+			$classes .= ' has-help-tabs';
+		}
+		?>
+		<div class="<?php echo esc_attr( $classes ); ?>">
+			<?php $this->render_notice_header(); ?>
+			<?php $this->render_notice_first_step(); ?>
+			<?php $this->render_notice_second_step(); ?>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Enqueue CSS for the admin bar.
+	 *
+	 * @return void
+	 */
+	public function enqueue_admin_bar_css() {
+		wp_enqueue_style(
+			'jetpack-idc-admin-bar-css',
+			plugins_url( '_inc/build/jetpack-idc-admin-bar.css', JETPACK__PLUGIN_FILE ),
+			array( 'dashicons' ),
+			JETPACK__VERSION
+		);
+	}
+
+	/**
+	 * Enqueue scripts for the notice.
+	 *
+	 * @return void
+	 */
+	public function enqueue_idc_notice_files() {
+		$build_assets                   = require_once __DIR__ . '/../build/index.asset.php';
+		$build_assets['dependencies'][] = 'jquery';
+
+		wp_enqueue_script(
+			'jetpack-idc-js',
+			Assets::get_file_url_for_environment( '_inc/build/idc-notice.min.js', '_inc/build/idc-notice.js' ),
+			$build_assets['dependencies'],
+			JETPACK__VERSION,
+			true
+		);
+
+		wp_localize_script(
+			'jetpack-idc-js',
+			'idcL10n',
+			array(
+				'apiRoot'         => esc_url_raw( rest_url() ),
+				'nonce'           => wp_create_nonce( 'wp_rest' ),
+				'tracksUserData'  => Jetpack_Tracks_Client::get_connected_user_tracks_identity(),
+				'currentUrl'      => remove_query_arg( '_wpnonce', remove_query_arg( 'jetpack_idc_clear_confirmation' ) ),
+				'tracksEventData' => array(
+					'isAdmin'       => current_user_can( 'jetpack_disconnect' ),
+					'currentScreen' => self::$current_screen ? self::$current_screen->id : false,
+				),
+			)
+		);
+
+		if ( ! wp_style_is( 'jetpack-dops-style' ) ) {
+			wp_register_style(
+				'jetpack-dops-style',
+				plugins_url( '_inc/build/admin.css', JETPACK__PLUGIN_FILE ),
```

Comment:
I'm not quite sure what to do in the case of this admin.css file.
Should I copy over the whole file and maybe rename it as jetpack-idc-admin.scss?